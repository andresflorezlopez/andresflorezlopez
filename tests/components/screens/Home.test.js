import React from "react";
import renderer from "react-test-renderer";
import { Provider } from "react-redux";

import Home from '../../../src/components/screens/Home'

import App from '../../../src/lib/app';

beforeEach(() => {
  App.boot().withState();
});

describe("Home component", () => {
  test("snapshot renders", () => {
    const component = renderer.create(
      <Provider store={App.store.redux}>
        <Home />
      </Provider>
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});