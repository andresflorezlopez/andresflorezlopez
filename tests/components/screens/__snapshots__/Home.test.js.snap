// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component snapshot renders 1`] = `
<div
  className="yfei84-0 iRwnoR"
>
  <header
    className="sc-1hlk1rh-0 kjLeSD"
  >
    <div
      className="sc-1kdsslr-0 dFURgt"
    >
      <ul
        className="sc-1hlk1rh-1 RuBqX"
      >
        <li
          className="sc-1hlk1rh-2 cUZCMh"
        >
          <div
            className="sc-1hlk1rh-3 NSkfu"
            to="/"
          >
            Rule of Thumb
          </div>
        </li>
        <li
          className="sc-1hlk1rh-2 cUZCMh"
        >
          <div
            className="sc-1hlk1rh-3 NSkfu"
            to="/about"
          >
            Past Trials
          </div>
        </li>
        <li
          className="sc-1hlk1rh-2 cUZCMh"
        >
          <div
            className="sc-1hlk1rh-3 NSkfu"
            to="/skills"
          >
            Past Trials copy
          </div>
        </li>
        <li
          className="sc-1hlk1rh-2 cUZCMh"
        >
          <div
            className="sc-1hlk1rh-3 NSkfu"
            to="/contact"
          >
            How It Works
          </div>
        </li>
      </ul>
    </div>
  </header>
  <div
    className="p68zvn-6 FMXTW"
  >
    <div
      className="p68zvn-0 fRxld"
    >
      <img
        alt="pope"
        className="p68zvn-1 foUsdE"
        src={Object {}}
      />
      <div
        className="sc-1kdsslr-0 dFURgt"
      >
        <div
          className="p68zvn-5 dabVwe"
        >
          <div
            className="p68zvn-7 hqFOyt"
          >
            What's your opinion on 
          </div>
          <div
            className="p68zvn-8 gyOJAN"
          >
            Pope Francis?
          </div>
          <div
            className="p68zvn-9 YymDO"
          >
            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed tellus urna, viverra a tempus eget, eleifend eget quam. Donec aliquam rhoncus mauris, tincidunt vulputate erat venenatis nec. Fusce congue felis eros, at malesuada sem lobortis a. 
          </div>
          <div
            className="p68zvn-10 kfFCOg"
          >
            W More information
          </div>
          <div
            className="p68zvn-11 eInMkd"
          >
            What's Your Veredict?
          </div>
          <div
            className="p68zvn-12 hEJDDn"
          >
            <div
              className="p68zvn-13 dFhWGv"
            />
            <div
              className="p68zvn-14 iHFoqC"
            />
          </div>
        </div>
      </div>
    </div>
    <div
      className="p68zvn-2 ezmJuV"
    >
      <div
        className="p68zvn-3 bQlRBc"
      >
        CLOSING IN
      </div>
      <div
        className="p68zvn-4 ixnmBr"
      >
        22 days
      </div>
    </div>
  </div>
  <div
    className="sc-1kdsslr-0 dFURgt"
  >
    <div
      className="vwnjkk-0 bRBQsP"
    >
      <div
        className="vwnjkk-1 hsKSso"
      >
        <div
          className="vwnjkk-2 kDisox"
        >
          Speak out. 
          <br />
          Be heard.
        </div>
        <div
          className="vwnjkk-3 ecTmWe"
        >
          Be counted.
        </div>
      </div>
      <div
        className="vwnjkk-4 kyGjPe"
      >
        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed tellus urna, viverra a tempus eget, eleifend eget quam. Donec aliquam rhoncus mauris, tincidunt vulputate erat venenatis nec.
      </div>
    </div>
  </div>
  <div
    className="sc-1kdsslr-0 dFURgt"
  >
    <div
      className="xtw21l-0 hEoPCl"
    >
      Previous Rulings
    </div>
    <div
      className="xtw21l-1 jdTpOL"
    >
      <div
        className="sc-66n42c-0 gaRFki"
      >
        <img
          alt="pope"
          className="sc-66n42c-6 epXrkH"
          src={Object {}}
        />
        <div
          className="sc-66n42c-1 bAwjXj"
        >
          <div
            className="sc-66n42c-2 hQwSUD"
          >
            Kaney West
          </div>
          <div
            className="sc-66n42c-5 iECHnN"
          >
            1 mont ago in Entertaiment
          </div>
          <div
            className="sc-66n42c-3 jOWpnj"
          >
            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed tellus urna, viverra a tempus eget, eleifend eget quam.
          </div>
        </div>
        <div
          className="sc-66n42c-4 jbnfKA"
        >
          <div
            className="sc-66n42c-8 bEdMLF"
            onClick={[Function]}
          >
            <svg
              aria-hidden="true"
              className="svg-inline--fa fa-thumbs-up fa-w-16 fa-2x "
              color="white"
              data-icon="thumbs-up"
              data-prefix="fas"
              focusable="false"
              role="img"
              style={Object {}}
              viewBox="0 0 512 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M104 224H24c-13.255 0-24 10.745-24 24v240c0 13.255 10.745 24 24 24h80c13.255 0 24-10.745 24-24V248c0-13.255-10.745-24-24-24zM64 472c-13.255 0-24-10.745-24-24s10.745-24 24-24 24 10.745 24 24-10.745 24-24 24zM384 81.452c0 42.416-25.97 66.208-33.277 94.548h101.723c33.397 0 59.397 27.746 59.553 58.098.084 17.938-7.546 37.249-19.439 49.197l-.11.11c9.836 23.337 8.237 56.037-9.308 79.469 8.681 25.895-.069 57.704-16.382 74.757 4.298 17.598 2.244 32.575-6.148 44.632C440.202 511.587 389.616 512 346.839 512l-2.845-.001c-48.287-.017-87.806-17.598-119.56-31.725-15.957-7.099-36.821-15.887-52.651-16.178-6.54-.12-11.783-5.457-11.783-11.998v-213.77c0-3.2 1.282-6.271 3.558-8.521 39.614-39.144 56.648-80.587 89.117-113.111 14.804-14.832 20.188-37.236 25.393-58.902C282.515 39.293 291.817 0 312 0c24 0 72 8 72 81.452z"
                fill="currentColor"
                style={Object {}}
              />
            </svg>
            <div
              className="sc-66n42c-10 gfgDKu"
            >
              10
              %
            </div>
          </div>
          <div
            className="sc-66n42c-7 eabijB"
            onClick={[Function]}
          >
            <div
              className="sc-66n42c-9 gsAjvy"
            >
              90
              %
            </div>
            <svg
              aria-hidden="true"
              className="svg-inline--fa fa-thumbs-down fa-w-16 fa-flip-horizontal fa-2x fa-pull-right "
              color="white"
              data-icon="thumbs-down"
              data-prefix="fas"
              focusable="false"
              role="img"
              style={Object {}}
              viewBox="0 0 512 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M0 56v240c0 13.255 10.745 24 24 24h80c13.255 0 24-10.745 24-24V56c0-13.255-10.745-24-24-24H24C10.745 32 0 42.745 0 56zm40 200c0-13.255 10.745-24 24-24s24 10.745 24 24-10.745 24-24 24-24-10.745-24-24zm272 256c-20.183 0-29.485-39.293-33.931-57.795-5.206-21.666-10.589-44.07-25.393-58.902-32.469-32.524-49.503-73.967-89.117-113.111a11.98 11.98 0 0 1-3.558-8.521V59.901c0-6.541 5.243-11.878 11.783-11.998 15.831-.29 36.694-9.079 52.651-16.178C256.189 17.598 295.709.017 343.995 0h2.844c42.777 0 93.363.413 113.774 29.737 8.392 12.057 10.446 27.034 6.148 44.632 16.312 17.053 25.063 48.863 16.382 74.757 17.544 23.432 19.143 56.132 9.308 79.469l.11.11c11.893 11.949 19.523 31.259 19.439 49.197-.156 30.352-26.157 58.098-59.553 58.098H350.723C358.03 364.34 384 388.132 384 430.548 384 504 336 512 312 512z"
                fill="currentColor"
                style={Object {}}
              />
            </svg>
          </div>
        </div>
      </div>
    </div>
  </div>
  <footer
    className="sc-1ekg0p-0 eIPBHH"
  >
    <div
      className="sc-1kdsslr-0 sc-1ekg0p-1 ivYvK"
    >
      <div
        className="sc-1ekg0p-2 bGUrnC"
      >
        <div
          className="sc-1ekg0p-3 hsjOPn"
        >
          Terms and Conditions
        </div>
        <div
          className="sc-1ekg0p-3 hsjOPn"
        >
          Politics
        </div>
        <div
          className="sc-1ekg0p-3 hsjOPn"
        >
          Contact us
        </div>
      </div>
      <div
        className="sc-1ekg0p-4 bPILXs"
      >
        <div
          className="sc-1ekg0p-3 hsjOPn"
        >
          Follow us:
        </div>
      </div>
    </div>
  </footer>
</div>
`;
